tutorials/algorithms/05_qaoa.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #47.    qaoa = QAOA(Sampler(), optimizer, reps=2)
</pre><p>Shouldn't the <code>sampler</code> variable defined in this cell go here instead of <code>Sampler()</code>?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>